service_client/helpers.py

```diff
@@ -0,0 +1,35 @@
+"""Helper functions for the service client."""
+
+from typing import cast
+from assisted_service_client import models
+
+
+class Helpers:
+    """Helpers contains functions that can be used by the service client."""
+
+    @staticmethod
```

Comment:
Not a python expert, but what's the upside of a class with static methods vs bare functions?

Reply:
Not a python expert either 😅 but from the way imports are written, I thought it would be cleaner to have all the methods under a class

Without a class, you'd have to import each function separately, and if we plan on adding more, I thought it might become cumbersome?

```
from service_client.helpers import get_platform_model, validate_vips, etc. 
```

Definitely can change it if there's good reason to!

Reply:
Yeah, that's a good reason, fine with me.